nodeadm/api/v1alpha1/nodeconfig_types.go

```diff
@@ -74,3 +75,25 @@ type ContainerdOptions struct {
 	// by the default configuration file.
 	Config string `json:"config,omitempty"`
 }
+
+// InstanceOptions determines how the node's operating system and devices are configured.
+type InstanceOptions struct {
+	LocalStorage LocalStorageOptions `json:"localStorage,omitEmpty"`
+}
+
+// LocalStorageOptions control how [EC2 instance stores](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html)
+// are used when available.
+type LocalStorageOptions struct {
+	Strategy LocalStorageStrategy `json:"strategy,omitEmpty"`
+}
+
+// LocalStorageStrategy specifies how to handle an instance's local storage devices.
+type LocalStorageStrategy string
+
+const (
+	// LocalStorageRAID0 will create a single raid0 volume from any local disks
+	LocalStorageRAID0 LocalStorageStrategy = "RAID0"
```

Comment:
This was actually added to the tool last week: https://github.com/elastic/crd-ref-docs/commit/cf959ab94ea543cb8efd25dc35081880b7ca6a81

The generated doc is super helpful but I assume this will improve over time